src/components/FoodBox.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import FoodItem from './FoodItem'
+
+export default class FoodBox extends Component {
```

Comment:
since there is no state here to manage, this could be a function component. This could even be implemented directly in App.js without the need of a component.